### ISSUE TITLE
Incoherent image loading

### ISSUE THREAD

**Issue by Auhmaan** [luberda-molinet/FFImageLoading#380]
Given a valid web url to an image, the `ImageViewAsync` doesn't load the image on a `Fragment` but - given the fact that in this case, this first `ImageViewAsync` links to another Fragment that contains yet another `ImageViewAsync` - it loads on another.

I've set up logging on the `Success`, `Error` and `Finish` and, in the first case, only the `Finish` is accessed whilst in the second goes through `Success` and `Finish`.

Is there any reason for this to happen? I've checked and the internet connection seems fine and the url is live ( I use the same url in both `ImageViewAsync`, hence the fact that it's weird to not load on the first but load on the second ).

**Comment by daniel-luberda** [comment]
Please configure custom logging as here: https://github.com/luberda-molinet/FFImageLoading/blob/master/samples/ImageLoading.Forms.Sample/Droid/MainActivity.cs#L21-L51 and copy your output here or make a reproduction. I need more details to help you.

**Comment by Auhmaan** [comment]
Unfortunately, and for now, I'm unable to test what could be the problem, since it started working. I added the snippet you provided on the code so if it happens on the future I'll add the log here. I'll later try to revert the changes to test the behavior with the snippet.

I might add that this might be problems with multiple loading of fragments. The `ImageViewAsync` that isn't loading the image properly is on a `Fragment` that is loaded at the same time as other `Fragment loads`.

Basically, what I'm doing is: on the `OnResume()` of an `Activity`, I load a `Fragment`, but check for a certain condition that if it's true, loads yet another `Fragment` on top of the previous one. The later is the one that contains the `ImageViewAsync` that doesn't load.

**Comment by daniel-luberda** [comment]
@Auhmaan Ok. Thanks for the info. I'll close it, feel free to reopen if you experience this once again. Thanks
